architecture/verifier/README.md

```diff
@@ -0,0 +1,83 @@
+# Verifier Refactor Changes Overview
+
+## Configuration
+
+Configuration for each Veraison component is encapsulated by a `ParamStore`
+that contains parameter definitions. A parameter definition specifies the type
+of the parameter (used in validation), and a path where it can be found within
+configuration.
+
+Configuration for a Veraison deployment is handled by a `Config` that gathers
+`ParamStore`s for components used in that deployment. The `Config`uses
+[Viper](https://github.com/spf13/viper) to gather configuration from various
+sources and populate `ParamStore`s associated with it. Parameters are populated
+based on their paths,  and the same value can be used to populate parameters
+inside multiple stores, if they share a path.
+
+<p/>
+<p align="center">
+  <img src="./veraison-config.png"/>
+</p>
+
+`ParamStore`s are created, populated with parameter definitions for their
+respective component, and then passed onto the `Config` that assumes ownership
+of them, and populates them from external sources. This happens on deployment
```

Comment:
So does line 24 imply that that ParamStore is re-populated?

Reply:
No, the `Config` populates them with actual values. The text above take about populating them with parameter definition.

I can see how this is confusing. Will reword.

Reply:
Thanks!